Gemfile

```diff
@@ -59,6 +59,8 @@ gem "faker"
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem "debug", platforms: %i[mri mingw x64_mingw]
+  gem 'factory_bot_rails', "~> 6.2.0"
```

Comment:
you have different quotation marks, run rubocop before submiting PR.
Run `rubocop -a` to auto fix some issues.

Reply:
Usually I prefer double quotes so I disabled "StringLiterals" in .rubocop.yml. Changed that so it only skips the files I want it :).